### PR TITLE
[wip]: Meta tags

### PR DESCRIPTION
**Pull request by katerynabugaieva**
General: 
Screenshot 2020-08-13 at 16 06 09

Facebook:
Screenshot 2020-08-13 at 17 17 52

Twitter
Screenshot 2020-08-13 at 18 16 55

LinkedIn 
Screenshot 2020-09-03 at 13 28 21

Xing
Screenshot 2020-09-03 at 13 28 55

*TODO*:
Whatsapp:
-add picture in preview
Screenshot 2020-08-13 at 16 07 54

### CODE REVIEW

File: src/components/seo.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ interface SeoProps {
   description?: string;
   lang?: string;
   meta?: [any];
```
Comment (feedm3) [inline src/components/seo.tsx]: I guess we can remove this property?

File: src/components/seo.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ interface SeoProps {
...
+  image?: string;
```
Comment (feedm3) [inline src/components/seo.tsx]: Better call it `imageSrc`, makes it a bit more clear

File: src/templates/blog-post.tsx [hunk 3]
```diff
@@ -35,12 +35,17 @@ const BlogPostTitle = styled(SectionTitle)`
...
+  console.log(data.markdownRemark.frontmatter.image);
   return (
     <Layout
       heroImage={data.markdownRemark.frontmatter.image}
       siteTitleUrl={'/blog'}
     >
-      <SEO title="Blog article" />
+      <SEO
+        title={data.markdownRemark.frontmatter.title}
+        image={data.markdownRemark.frontmatter.image}
```
Comment (feedm3) [inline src/templates/blog-post.tsx]: use the excerp from the markdown as description. you can check the `blog.tsx` for more information

**Review by feedm3 — APPROVED** [review]

File: src/components/seo.tsx (hunk 1 above)
Comment (katerynabugaieva) [reply src/components/seo.tsx]: was solved with next commits

File: src/components/seo.tsx [hunk 4]
```diff
@@ -84,6 +84,9 @@ const SEO: React.FC<SeoProps> = ({
       )}
       {imageUrl && <meta property="og:image:width" content="300" />}
 
+      {/* -- Xing --*/}
+      <meta property="og:site_name" content={metaDescription} />
```
Comment (feedm3) [inline src/components/seo.tsx]: `site_name` should be the `title`. `description` is too long

But still nice that you add the custom description as property, as it's already used on `twitter:desription` for example!

**Review by feedm3 — CHANGES_REQUESTED** [review]

**Review by feedm3 — APPROVED** [review]

**Pull request merged**